### DISCUSSION TITLE
Abstürze im Setup / Crashes while in Setup

### DISCUSSION THREAD

**Discussion by Zeench** [jomjol/AI-on-the-edge-device#771] [Q&A]
Hi,

ich habe hier 2 Module geflasht.

Im Setup komme ich max. bis zum Schritt analoge Einstellung, dann stürzt das Ganze ab.

Das ist bei beiden Modulen so.
Habe das Wiki und diverse Youtube Videos angeschaut und kann keinen Fehler in meinem Vorgehen finden.
Ist es möglich das Beide Module "kaputt" sind?

Was auffällt ist, dass ich das Setup, nach ein paar Sekunden, neustarten kann und es auch reagiert.
Sobald aber ein Bild der Kamera geladen wird, also aus dem Speicher, stürzt das ganze in 99% Fälle ab.
einmal bin ich bis zum Schritt analog gekommen, aber nun nicht mehr. :(

Ich möchte das so gerne ans laufen bekommen. :)
Bin für jede Hilfe dankbar.

Gruß
Christoph

---------------------------English below:

Hi,

I have flashed 2 modules here.

In the setup I get max. to the step analog setting, then the whole thing crashes.

This is the case with both modules.
Have looked at the wiki and various Youtube videos and can not find any error in my procedure.
Is it possible that both modules are "broken"?

What is noticeable is that I can restart the setup, after a few seconds, and it also responds.
But as soon as a picture of the camera is loaded, from the memory though, I got to the step analog once, but now no more. :(

I would like to get this running. :)
I am grateful for any help.

Greetings
Christoph

**Comment by jomjol** [comment]
Kannst du mal das Logfile der seriellen Schnittstelle während des Startvorgangs posten?

**Reply by Zeench** [reply]
Du meinst wieder vom wasserzähler abbauenan den PC anschließen?
Wie erzeuge ich dann ein logfile?

**Comment by jomjol** [comment]
ja genau und den Ausgang der seriellen Schnittstelle mit loggen. Es spielt keine Rolle, dass das Bild nichts sinnvolles enthält.

**Reply by Zeench** [reply]
Ich weiss aber nicht wie das geht :(

Kannste mir mal auf die Sprünge helfen?

Danke schon jetzt für deine Hilfe :)

**Comment by jomjol** [comment]
Da ich nicht mal dein Betriebssystem kenne, ist das schwierig. Da findest du mit sicherheit duzende Anleitung sehr einfach mit googeln :-)

**Reply by Zeench** [reply]
Habe Win10.
Habe jetzt etliche Websites dazu gelesen, bin zu doof Putty zum loggen zu bringen.
Bleibt immer leer oder es kommen kryptische Zeichen. 
Kann den COM5 Port (ist bei mir der USB Port wo ich den ESP32 dran hänge) auch erst öffnen wenn ich den ESP 32 angeschlossen habe, sonst kommt eine Fehlermeldung. Vermutlich weil der Port bis dahin nicht existiert
![image](https://user-images.githubusercontent.com/74009385/163991618-89269ee7-8bab-4296-9153-16cf036114e0.png)

ABER:
Ich habe es jetzt so oft versucht und immer schön die config.ini bei jedem neuen Step gespeichert.
Habe also das Setup abschließen können.
Funktioniert auch soweit, nur das wie gesagt ständig Abstürze kommen sobald ich auf der Weboberfläche irgendwas machen möchte.
Egal was, manchmal passiert es schon wenn ich nur den config reiter öffnen will, ein anderes mal wenn ich den Pre Value setzen wollte, etc.
Für mich nicht reproduzierbar. Passiert willkürlich, aber stetig.

Solange ich die Weboberfläche nicht nutze funktioniert alles tadellos.
Es wird per MQTT ein Topic zum iobroker geschickt und ich kann das in meine Influxdb schreiben.
Aber wie gesagt sobald ich auf der Webpberfläche irgendetwas machen will, ist es wie beim Glücksspiel ob es funktioniert oder er abstützt. 

Es läuft nun zwar aber zufriedenstellend ist das auch nicht. :(

**Comment by jomjol** [comment]
Das klingt eher nach einem Netzwerkthema - inkompatibler Brower, Firewallblockade, etc. Denn wenn der MQTT etwas liefert, dann laufen eigentlich alle kritischen Teile des ESP32 und es kann nur an der Übertragung zwischen ESP32 und deinem Web-Browser liegen.

**Reply by Zeench** [reply]
Also getestet habe ich es mit Edge, Chrome, Firefox, und mobil.
Bei allen tritt das auf.

Firewall gibt es keine.

Wie gesagt der ESP arbeitet ja, aber sobald ich z.b. die Erkennungsmarken neu sortieren möchte und er den "Reiter" mit dem Bild laden will, passiert gar nix mehr, es ist dann auch nichts mehr auswählbar.
Nach 5Sek kann ich dann einen neuen Tab/Fenster öffnen und den ESP wieder erreichen.
Das ist übrigens in der Tat komisch, ein einfaches drücken von F5 zum aktualisieren der Seite funktioniert nicht.
Ich muss wirklich ein neues Fenster öffnen um neu rein zu kommen.

Mich würde das Putty log doch sehr interessieren, aber ich bin wirklich zu doof dafür :(
Danke erstmal für deine Mühe und Zeit @jomjol

**Comment by jomjol** [comment]
Dann ist das Fehlerbild ziemlich einzigartig. So etwas habe ich noch nicht gehört. Das läßt allerdings remote auch quasi kaum debuggen.
Beobachtung:
- Web-Server läßt sich quasi nicht bedienen, da andauernd die Verbindung abbricht und eine neue Verbindung erfordert
Fakten:
- keine Firewall,
- kein Virenscanner
- tritt bei allen unterschiedlichen Browsern auf - auch Firefox, der auf anderen System ohne derartige Schwierigkeiten tut.

Einzige Ursache, die ich mir noch vorstellen könnte, wäre ein Reboot bei Zugriff auf die HTML-Seite. Was das allerdings triggert wäre mir ein Rätsel. Ich vermute, die SD-Karte ist auch okay.

**Reply by tompsg-git** [reply]
Ich hatte das gleiche Phänomen. Nach dem Neuaufspielen der sdcard-Daten waren alles in Ordnung. Evtl. ein Dateisystemfehler.
